R/utils.R

```diff
@@ -498,6 +502,10 @@ end_year_lookup <- function(scenario_type) {
     end_year <- as.numeric(2100)
   }
 
+  if (scenario_type == "is_oxford") {
```

Comment:
improvement: you could combine all scenario types that have the same end-year in 1 condition a la if (scenario_type in c(a,b,c)) to make it easier to read